misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,42 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  fun getDefaultBackoff(): ExponentialBackoff {
+    return ExponentialBackoff(
+        baseDelay = Duration.ofMillis(10),
+        maxDelay = Duration.ofMillis(500),
+        jitter = Duration.ofMillis(5)
+    )
+  }
+
+  /**
+   * Retries any [AmazonDynamoDBException] or [PleaseRetryException]. [DontRetryException] will
+   * not be retried.
+   */
+  fun <T> dynamoDbRetry(
+    retries: Int = 3,
+    backoff: Backoff = getDefaultBackoff(),
```

Comment:
I think this is still incorrect: `getDefaultBackoff()` gets evaluated once statically and is shared as a global variable

Reply:
I don't follow. Isn't `getDefaultBackoff()` called every time `dynamoDbRetry` is called? Are the default value of parameters static?